dashboard-app/src/features/dashboard/Dashboard/useDashboardStatistics.ts

```diff
@@ -205,6 +210,8 @@ export const timeStatsToProps = (pts?: EqualDataPoints) => {
     left: {
       label: leftLabel,
       data: pts.labels,
+      limit: 3,
+      placeholder: '...',
```

Comment:
Placeholder is a misleading name. This does not describe what this value is doing.